filebeat/module/nginx/ingress_controller/ingest/pipeline.yml

```diff
@@ -41,6 +61,81 @@ processors:
     field: source.address
     if: ctx.source?.address == null
     value: ""
+- script:
+    if: ctx.nginx?.ingress_controller?.upstream?.response?.length_list != null && ctx.nginx.ingress_controller.upstream.response.length_list.length > 0
+    lang: painless
+    source: >-
+      try {
+        if (ctx.nginx.ingress_controller.upstream.response.length_list.length == null) {
+          return;
+        }
+        int length = 0;
+        for (def item : ctx.nginx.ingress_controller.upstream.response.length_list) {
+          length = length + Integer.parseInt(item);
```

Comment:
Hmmm, does it make sense to sum up the response lengths? The client would only see the most recent response.

Reply:
That makes sense, I will go with it! Thanks!